src/views/about.ejs

```diff
@@ -0,0 +1,246 @@
+<% include partials/header %>
+<div class="body">
```

Comment:
body tag is already declared inside the `header.ejs` file above. Kindly verify this.